src/jspin.c

```diff
@@ -30,6 +30,10 @@
  */
 bool jshIsPinValid(Pin pin) {
   // Note, PIN_UNDEFINED is always > JSH_PIN_COUNT
+  if ((pin < JSH_PIN_COUNT && pinInfo[pin].port != JSH_PORT_NONE) == 0) {
```

Comment:
This doesn't seem right at all. `jshIsPinValid` is a check - it shouldn't produce an error message, and it's used in IRQs where writing to the console could potentially lock the system up. You could create a second function that did produce an error message if you needed to?